Resources/views/full/folder.html.twig

```diff
@@ -0,0 +1,50 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    <section class="content-view-full">
+        <div class="class-folder">
+            <div class="row">
+                <div class="col-md-8">
+                    <div class="attribute-header">
+                        <h1>{{  ez_render_field( content, 'name' ) }}</h1>
+                    </div>
+
+                    {% for article in pagerFolder %}
+                        <section class="content-view-children">
+                            {{ render_esi( controller( 'ez_content:viewLocation', {'locationId': article.contentInfo.mainLocationId, 'viewType': 'line'} ) ) }}
+                        </section>
+                    {% endfor %}
+
+                    {% if pagerFolder.haveToPaginate() %}
+                        <div class="pagination-centered">
+                            {{ pagerfanta( pagerFolder, 'twitter_bootstrap_translated', {'routeName': location} ) }}
+                        </div>
+                    {% endif %}
+
+                </div>
+                <div class="col-md-4">
+                    <aside>
+                        <section class="subnavigation row nav-collapse">
+                            <div class="attribute-header">
+                                <h2>{{  ez_render_field( content, 'name' ) }}</h2>
+                            </div>
+
+                            {% if subContent|length > 1 %}
+                                <ul class="break">
+                                    {% for entry in subContent %}
+                                        {% if entry.contentInfo.mainLocationId != location.contentInfo.mainLocationId %}
+                                            <li>
+                                                <a href="{{ path( 'ez_urlalias', { 'locationId': entry.contentInfo.mainLocationId } ) }}" title="{{ ez_field_value( entry, 'name' ) }}">{{ ez_field_value( entry, 'name' ) }}</a>
+                                                {{ render_esi( controller( 'eZDemoBundle:Demo:showSubFolderLink', { 'locationId': entry.contentInfo.mainLocationId } ) ) }}
+                                            </li>
+                                        {% endif %}
+                                    {% endfor %}
+                                </ul>
+                            {% endif %}
+                        </section>
+                    </aside>
+                </div>
+            </div>
+        </div>
+    </section>
+{% endblock %}
```

Comment:
It's a commentabout the whole file. I don't like the HTML code structure. Where can I see the final result?